docs/source/guide/label_object.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Retrieve imported data in the labeling configuration
```

Comment:
This title is bad.. Actually we have this section - https://deploy-preview-5527--label-studio-docs-new-theme.netlify.app/guide/label_components#Variables - and it's very related to this whole page. 

This title should be like "Variable values and types"

Reply:
The long title can be "Retrieve imported data in the labeling configuration" => "Variable values and value types: different ways of data loading in the labeling configuration"